CONTRIBUTING.md

```diff
@@ -89,3 +89,27 @@ The tool will replace the text with the following:
 ```markdown
 See [§19.23.42](features.md#19-23-42-new-feature-added) for more details on how this feature affects everything. The [§19.23.42.1](features.md#19-23-42-1-general) clause has an overview.
 ```
+
+## Coding style
+
+The code in the standard is optimized for reading the standard. In general, follow the existing code format. We've tried to be consistent throughout the standard.
+
+We've adopted the following non-exhaustive rules:
+
+- Use the latest C# syntax for the branch being edited. This is currently C# 7.3, except where older syntax is required for the example. Readers of the standard should see the latest syntax.
+- Use [Allman style braces](https://en.wikipedia.org/wiki/Indentation_style?msclkid=5bd8d8f5cd7b11ec82461327ced4546a#Allman_style) in most of our samples. The exception is when a method or type is empty, where the declaration, the opening and closing brace are on the same line.
+- Use a blank line between peer declarations. There is two exceptions to this rule:
+  - If a group of peer declarations all have empty bodies (e.g. `class C {}` or `public void M() {}`, then no blank line is added between them.
+  - Don't add a blank line where the concept is better illustrated without them. For example, multiple overloads of the same method may read better grouped without a blank line.
+- Elements of the same type can be grouped without blank lines. When that's done, there should be a blank line between elements of different types. (e.g. all private fields can be declared without a blank line, but there should be a blank line between the group of private fields and method declarations).
+- Indentation is 4 spaces. Use spaces, not tabs for the Word and PDF export formats.
+- Where a group of lines all have comments appended, the opening `//` should be on the same column on all lines.
+- Where expression bodied members are used, place all code on the same line if it fits. If it's too long, wrap after the `=>` token.
+- Line lengths cannot exceed 80 characters.
```

Comment:
Jon's earlier PR for line length used 95 characters. The word converter has a warning at 80 characters, so I used that instead.

Reply:
Yup - I had to change from 95 to 80 due to a change in the Word font.